components/ChipTextField/examples/ChipTextFieldExampleViewController.swift

```diff
@@ -0,0 +1,128 @@
+// Copyright 2018-present the Material Components for iOS authors. All Rights Reserved.
```

Comment:
Please update the copyright headers here and throughout.

Reply:
I believe this code was originally authored in 2018. It was rolled-back during the Jan 16 release due to internal breakages. It's now being partly "rolled forward."